tests/functional/pv/pv_services/test_cephfs_with_mds_network_failure.py

```
@@ -66,31 +67,39 @@ def test_cephfs_network_interruption_standby_replay_MDS(self, pod_factory, reque
         )
         io_thread.start()
 
-        # Cause network interruption to the standby-replay daemon three times
-        log.info("Starting network interruptions...")
+        logger.test_step("Cause 3 network interruptions to standby-replay MDS node")
+        logger.info(
+            f"Starting network interruptions on node {self.standby_replay_node_ip}"
+        )
         for i in range(1, 4):
```

Ruff (0.15.15)

[warning] Loop control variable `i` not used within loop body

Rename unused `i` to `_i`

(B007)